lib/queuePopulator/LogReader.js

```diff
@@ -381,6 +390,35 @@ class LogReader {
         return undefined;
     }
 
+    _processFilterEntries(batchState, done) {
+        const { entriesToPublish, currentRecord } = batchState;
+        if (!currentRecord || !currentRecord.entries) {
+            return done();
+        }
+        async.series([
```

Comment:
There is no need to use `async.series` as there's only one truly async call in the list (the 2nd one)